README.md

```diff
@@ -1,27 +1,13 @@
-# Arrow Contracts
-
-Implementation of the Arrow token and governance smart contracts.
+# Arrow Contract
 
 ## Installation
 
-1. [Install Brownie](https://eth-brownie.readthedocs.io/en/stable/install.html), if you haven't already.
-
-## Testing
-
-To run the tests:
-
-```bash
-brownie test
-```
-
-## Deployments
+Clone the repo and then run `npm install` to get started.
 
-### Rinkeby
+## Deploy contracts
 
-| Contract | Address |
-| -- | -- |
-| Gnosis Safe | 0x6943eBEfCD7d85B536aEb35BBFd95C5699158Abe |
```

Comment:
We shouldn't remove contract addresses for existing contracts. These already exist on the testnet and haven't been updated.